idb/postgres/postgres.go

```diff
@@ -177,78 +176,6 @@ func (db *IndexerDb) init(opts idb.IndexerDbOptions) (chan struct{}, error) {
 	return db.runAvailableMigrations(opts)
 }
 
-// Preload asset and app creators.
-func prepareCreators(l *ledger_for_evaluator.LedgerForEvaluator, payset transactions.Payset) (map[basics.AssetIndex]ledger.FoundAddress, map[basics.AppIndex]ledger.FoundAddress, error) {
```

Comment:
These are being deleted by #1095 
**DON'T MERGE AHEAD OF THAT PR AND MAKE SURE THERE ARE NO DIFFS HERE**